### PR TITLE
fix(client): force numeric input for age field on signup page

### LINKED ISSUE #3639: FxA can be created using special characters in the age field [bz1260731]
From https://github.com/mozilla/fxa-bugzilla-mirror/issues/134 which was copied from
From https://bugzilla.mozilla.org/show_bug.cgi?id=1260731

Build: 3.0 (9)

Steps to reproduce:
1. Launch Mozilla Firefox
2. Create Firefox account
3. Type "1." or "12," in the age field

Actual result:
- The user can create an account with an incorrect inputted age

Expected result:
- The user age field should not contain special characters 

Note: 
- Typing "."/"," characters in the age field will cause inconsistency regarding the minimum age required to create Firefox account.
- This inconsistency does not occur on Android version as the age field will become blank when typing these characters.
- When typing other characters like ")[%;" the age text field is deleted.

### PR DESCRIPTION
**Pull request by vladikoff**
Fixes #3639

### CODE REVIEW

**Comment by vladikoff** [comment]
@shane-tomlinson please r?

File: app/scripts/views/coppa/coppa-age-input.js [hunk 1]
```diff
@@ -56,6 +56,22 @@ define(function (require, exports, module) {
       age.val(age.val().substr(0, 3));
     },
 
+    onKeyDown: function (event) {
+      // helper function to check for digit
+      function isKeyADigitOrBackSpace (keyCode) {
+        return ((keyCode === KeyCodes.BACKSPACE) ||
+        (keyCode >= KeyCodes.NUM_0 && keyCode <= KeyCodes.NUM_9) ||
+        (keyCode >= KeyCodes.NUMPAD_0 && keyCode <= KeyCodes.NUMPAD_9));
```
Comment (shane-tomlinson) [inline app/scripts/views/coppa/coppa-age-input.js]: I just realized through testing that this no longer allows the arrow keys to be used to move through the text input.

**Comment by shane-tomlinson** [comment]
@vladikoff - other than the arrow keys, looks great!

**Comment by vladikoff** [comment]
@vbudhram hey could you review the last small commit https://github.com/mozilla/fxa-content-server/pull/3803/commits/0e33a7cc0da8111a901f7b1878edf33bc2ad5f50 that fixes: https://github.com/mozilla/fxa-content-server/pull/3803#discussion_r65519506

**Comment by vbudhram** [comment]
LGTM, pending squishing commits, r+

**Pull request merged**